WADE.R

```diff
@@ -71,13 +71,13 @@ ui <- fluidPage(
       conditionalPanel(condition = "input.Org == 'GONO'",
                        radioButtons("test", h4("Choose an analysis:"),
                        choices = list("AMR Profile" = "AMR_ALL",
-                                      #"AMR Alleles" = "AMR",
+                                      "AMR Alleles" = "AMR",
                                       "23S rRNA Alleles" = "rRNA23S",
                                       "MLST" = "MLST",
                                       "NG-STAR" = "NGSTAR",
                                       "NG-MAST" = "NGMAST",
                                       "All Routine Analyses" = "ALL",
-                                      "MasterBlastR" = "MASTERBLASTR",
+                                      #"MasterBlastR" = "MASTERBLASTR",
```

Comment:
Was commenting this out a part of what was needed for Linux compatibility or was it for a different change?

Reply:
This is a function we just use for research so I took it out of the github version

Reply:
Ah, okay, makes sense. I think everything looks good then 😄